src/docfx/lib/git/GitCommitProvider.cs

```diff
@@ -244,84 +248,84 @@ private unsafe (List<Commit>, Dictionary<long, Commit>) LoadCommits()
                 if (error != 0)
                     throw new InvalidOperationException($"Unknown error calling git_revwalk_next: {error}");
 
-                GitObjectLookup(out var commit, _repo, &commitId, GitObjectType.Commit);
-                var author = GitCommitAuthor(commit);
-                var committer = GitCommitCommitter(commit);
-                var parentCount = GitCommitParentcount(commit);
-                var parents = new GitOid[parentCount];
+                git_object_lookup(out var commit, _repo, &commitId, 1 /* GIT_OBJ_COMMIT */);
```

Comment:
why not using enum GitObjectType any more?

Reply:
We only use 2 of them, there are various other enums we use that are not defined, like `GIT_ITEROVER`, I just use this style to be consistent.